addons/mail/static/src/components/message/message.xml

```diff
@@ -18,7 +18,7 @@
                 <div class="o_Message_sidebar" t-att-class="{ 'o-message-squashed': props.isSquashed }">
                     <t t-if="!props.isSquashed">
                         <div class="o_Message_authorAvatarContainer o_Message_sidebarItem">
-                            <img class="o_Message_authorAvatar rounded-circle" t-att-class="{ o_Message_authorRedirect: hasAuthorOpenChat, o_redirect: hasAuthorOpenChat }" t-att-src="avatar" t-on-click="_onClickAuthorAvatar" t-att-title="hasAuthorOpenChat ? OPEN_CHAT : ''" alt="Avatar"/>
+                            <img class="o_Message_authorAvatar rounded-circle" t-att-class="{ o_Message_authorRedirect: hasAuthorOpenChat, o_redirect: hasAuthorOpenChat }" t-att-src="avatar" t-on-click="_onClickAuthorAvatar" t-att-title="hasAuthorOpenChat ? OPEN_CHAT : ''" t-att-alt="hasAuthorOpenChat ? OPEN_CHAT : ''" role="button" tabindex="0"/>
```

Comment:
This change looks wrong, shouldn't `alt` describe the image? Not the action when clicking on it.

Reply:
That would be the case if the element were acting as an image, but in this case, it's acting as a button, so button action would be the correct text. If we were going to describe the image, a text like "rounded circle" should be used, which gives no idea what's the button for.

Take into account the alt attribute is the equivalent as `aria-label` but for image tags, so alt is used because of the tag, not the text's purpose.